view.php

```diff
@@ -221,16 +221,25 @@
 $table->size = array('35%', '65%');
 $numcolumns = 2;
 
-// Show start/end date or recurring flag.
-if ($zoom->recurring) {
+// Show start/end date or recurring meeting information.
+if ($zoom->recurring && $zoom->recurrence_type == ZOOM_RECURRINGTYPE_NOTIME) {
     $table->data[] = array(get_string('recurringmeeting', 'mod_zoom'), get_string('recurringmeetingexplanation', 'mod_zoom'));
+} else if ($zoom->recurring && $zoom->recurrence_type != ZOOM_RECURRINGTYPE_NOTIME) {
+    $table->data[] = array(get_string('recurringmeeting', 'mod_zoom'), get_string('recurringmeetingthisis', 'mod_zoom'));
+    if (($nextoccurrence = zoom_get_next_occurrence($zoom)) > 0) {
```

Comment:
Trying to minimize assignments within conditionals.
```suggestion
    $nextoccurrence = zoom_get_next_occurrence($zoom);
    if ($nextoccurrence > 0) {
```

Reply:
have updated that

Reply:
I must be missing a commit, because I don't see this one either.